packages/mui-joy/src/Accordion/AccordionContext.ts

```diff
@@ -8,6 +8,7 @@ const AccordionContext = React.createContext<
   Partial<{
     disabled: boolean;
     expanded: boolean;
+    type: string;
```

Comment:
```suggestion
```

I don't think type should be a part of the context here.

Reply:
comment resolved